bptestrunner/bluepill_batch_test.bzl

```diff
@@ -0,0 +1,142 @@
+load(
```

Comment:
Can you pass this through `buildifier --lint=warn --warnings=all`?

Reply:
ran 
find . -name *.bzl -exec buildifier -lint=fix {} \;